ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsAnalytics.kt

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.api
+
+internal object ShopperInsightsAnalytics {
+    const val PAYPAL_PRESENTED = "shopper-insights:paypal-presented"
+    const val PAYPAL_SELECTED = "shopper-insights:paypal-selected"
+    const val VENMO_PRESENTED = "shopper-insights:venmo-presented"
+    const val VENMO_SELECTED = "shopper-insights:venmo-selected"
```

Comment:
Not sure if it'll be a follow up PR but I thought I had heard we don't need to care about Venmo anymore for this feature. Totally fine to 🏈 on it if we are still waiting to get clarity, but just didn't want to add events if we already know Venmo will be removed.

Reply:
I not 100%, but we will likely have Venmo.
I can DM you the exact thing I heard in the meeting yesterday.

Reply:
Nah no worries - I trust you! You're more keyed into the going ons with this feature than I am. 😄